_posts/2023-09-01-pixel-mocking.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Cockpit 299
+author: mvo
+date: '2023-09-01'
+tags: cockpit
+slug: pixel-mocking
+category: tutorial
+summary: 'Mocking the DOM for Better Pixels, plus statistics'
+---
+
+When testing pixels, once needs to deal with parts of the UI that
```

Comment:
`once` -> one?

Reply:
Thanks! I should have made this draft, I didn't actually proof read this myself in the beginning...